conanfile.py

```diff
@@ -33,8 +33,8 @@ def config_options(self):
 
     def build_requirements(self):
         self.build_requires('yasm/1.3.0')
-        if tools.os_info.is_windows and "CONAN_BASH_PATH" not in os.environ:
-            self.build_requires("msys2/20161025")
+        if tools.os_info.is_windows and "CYGWIN" not in os.environ:
```

Comment:
How about add condition ("CONAN_BASH_PATH" not in os.environ) here? Without this test, cygwin_installer is always installed even user has already installed a bash environment.

Reply:
Because the test I did with msys2 failed, and documentation of libvpx says cygwin is needed (see links in the pull request's message). Do you kmow how to fix the build in msys environment?